inginious/frontend/templates/task_dispensers_admin/config_items/accessibility.html

```diff
@@ -83,21 +83,33 @@
     $("#edit_task_modal").on('show.bs.modal', function (event) {
         var button = $(event.relatedTarget);
         var taskid = button.data('taskid');
+
         if (!("accessibility" in dispenser_config[taskid]))
             return;
         var accessibility = dispenser_config[taskid]["accessibility"];
         var value;
 
-        if (accessibility === true) {
-            value = 'true';
-        } else if (accessibility === false) {
-            value = 'false';
-        } else {
+        special_dates = ["1-01-01 00:00:00", "9999-12-31 23:59:59"]
```

Comment:
We should print `datetime.min` or `datetime.max` (without the microseconds) instead, to ensure consistency.